public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -319,46 +326,60 @@ export default class FindingDetailsFlyout extends Component<
     return patternId;
   };
 
-  toggleDocumentDetails(item: FindingDocumentItem) {
-    const docIdToExpandedRowMapValues = { ...this.state.docIdToExpandedRowMap };
+  createDocumentBlock = (item: FindingDocumentItem) => {
     let formattedDocument = '';
     try {
       formattedDocument = document ? JSON.stringify(JSON.parse(item.document), null, 2) : '';
     } catch {
       // no-op
     }
 
+    return (
+      <EuiFormRow fullWidth={true} style={{ width: '100%' }}>
+        <EuiCodeBlock
+          language="json"
+          isCopyable
+          data-test-subj={`finding-details-flyout-rule-document-${item.itemIdx}`}
+        >
+          {formattedDocument}
+        </EuiCodeBlock>
+      </EuiFormRow>
+    );
+  };
+
+  getRelatedDocuments() {
```

Comment:
could improve code readability and maintainability ` getRelatedDocuments(): FindingDocumentItem[] `is a custom

Reply:
Let me send a quick follow up PR, sorry didn't pay full attention here